performance/streaming-optimization.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Streaming Optimizations"
+description: "RisingWave provides a variety of optimizations to improve the performance of streaming queries."
+sidebarTitle: Overview
+---
+
+This section outlines best practices for Streaming Optimizations.
+
+## SQL Optimizations
+
+### UNION vs UNION ALL
+
+UNION removes duplicates, while UNION ALL does not.
+If you know that your data does not contain duplicates, use UNION ALL to avoid the overhead of duplicate removal.
+Alternatively, if the duplicate set is small, you can also filter out duplicates when querying the result set.
+
+### OverWindow vs GroupTopN
```

Comment:
refine the page https://docs.risingwave.com/processing/sql/top-n-by-group , and link here instead?

Reply:
https://github.com/risingwavelabs/risingwave-docs/pull/401/commits/41f36204bcf667392b8410c2c55606bdddc32484